lib/gc.js

```diff
@@ -202,12 +199,7 @@ GarbageCollector.prototype = {
     }
   },
 
-  scheduleSweep: function (interval) {
-    this.sweepTimeoutId = setTimeout(this.sweep.bind(this), interval);
-  },
-
-  sweep: async function () {
-    clearTimeout(this.sweepTimeoutId);
+  sweep: async function (full=false) {
```

Comment:
In this case I just want things that care about a full sweep to do so or not, without needing to know exactly what a number represents.  It seems clearer that the caller of the sweep() method just say if it should perform a full one or not.